keyboards/cipulot/ec_pro2/readme.md

```diff
@@ -0,0 +1,19 @@
+# EC Pro2
+
+![EC Pro 2 PCB](https://i.imgur.com/uYOgTYo.png)
+
+HHKB Pro2 replacement PCB.
+
+* Keyboard Maintainer: [cipulot](https://github.com/cipulot)
+* Hardware Supported: EC Pro2 PCB
+* Hardware Availability: [Github](https://github.com/Cipulot/EC-Pro-2)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make cipulot/ec_pro2:default
+
+Flashing example for this keyboard:
+
+    make cipulot/ec_pro2:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Can you add some instructions here on entering bootloader mode, as per the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md). 

Otherwise looks good to me.

Reply:
Done 👍